src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
+
+User.propTypes = {
+  userData: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      phone: PropTypes.string.isRequired,
+      mail: PropTypes.string,
+      department: PropTypes.string.isRequired,
+    })
+  ),
+  skills: PropTypes.arrayOf(
+    PropTypes.shape({
+      skill1: PropTypes.string.isRequired,
+      skill2: PropTypes.string,
+      skill3: PropTypes.string,
+      skill4: PropTypes.string,
+      skill5: PropTypes.string,
+      skill6: PropTypes.string,
+    })
+  ),
+}
```

Comment:
You set `PropTypes` but your `User` doesn't  receive any props. The `users`-array you are working with, is an array created inside your `User`-component and not a property.